### ISSUE TITLE
The demo page is unavailable

### ISSUE THREAD

**Issue by hayschan** [mr-internetix/vcard-qr-code-generator#2]
The demo page of this repo is not available for access.

https://[vcard-qr-code.herokuapp.com](https://vcard-qr-code.herokuapp.com/)/

**Comment by mr-internetix** [comment]
yeah i know, you can run it on local by following the instructions in readme.md , actually heroku closed his free subscription tier, that's why demo page is not available.

if you still want the demo page you can comment it down !

regards,
mr_internetix
